packages/frontend/src/Build.svelte

```diff
@@ -684,14 +762,115 @@ $: if (availableArchitectures) {
               <!-- svelte-ignore a11y-no-static-element-interactions -->
               <span
                 class="font-semibold mb-2 block cursor-pointer"
-                aria-label="advanced-options"
-                on:click={toggleAdvanced}
-                ><Fa icon={showAdvanced ? faCaretDown : faCaretRight} class="inline-block mr-1" />Advanced Options
+                aria-label="build-config-options"
+                on:click={toggleBuildConfig}
+                ><Fa icon={showBuildConfig ? faCaretDown : faCaretRight} class="inline-block mr-1" />Build config
               </span>
-              {#if showAdvanced}
+              {#if showBuildConfig}
+                <p class="text-sm text-[var(--pd-content-text)] mb-2">
+                  Supplying the following fields will create a build config file that contains the build options for the
```

Comment:
It never says _where_ the build config will be generated. I think this would help (e.g. say we put it in the folder above), so that I could use this once to create it, then maybe edit it outside of Podman Desktop, then supply the same file next time I build.